src/pg/brc20/brc20-pg-store.ts

```diff
@@ -35,17 +36,27 @@ export class Brc20PgStore {
     this.parent = db;
   }
 
+  sqlOr(partials: postgres.PendingQuery<postgres.Row[]>[] | undefined) {
+    return partials?.reduce((acc, curr) => this.sql`${acc} OR ${curr}`);
+  }
+
   async getTokens(
     args: { ticker?: string[] } & DbInscriptionIndexPaging
   ): Promise<DbPaginatedResult<DbBrc20Token>> {
-    const lowerTickers = args.ticker ? args.ticker.map(t => t.toLowerCase()) : undefined;
+    const tickerPrefixCondition = this.sqlOr(
+      args.ticker?.map(t => this.sql`d.ticker_lower LIKE ${t.toLowerCase()} || '%'`)
```

Comment:
Several points here:
* We shouldn't compare `toLowerCase()` from JS with `LOWER` from postgres (my mistake originally) as they may yield different results
* Can you explain why you do `LIKE ${t.toLowerCase()} || '%'`? I think we should only let the ticker be autocomplete from a prefix instead of searching for any substring

Reply:
👍🏻 I'll wrap in LOWER

`LIKE abc%` is a prefix search, correct?